addons/rounded-thumbs/profile.css

```
@@ -0,0 +1,32 @@
+img {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
+
+.gallery.thumb span.image {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1.35px);
+}
+
+#profile-box .player {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
+
+.media-grid .thumb {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
+
+.gallery.thumb span.image img {
+  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
+}
```

```suggestion
#profile-box .player,
.media-grid .thumb,
.gallery.thumb span.image img {
  border-radius: calc(var(--roundedThumbs-roundSize) * 1px);
}
```
Styles could also be grouped elsewhere too to clean them up a little.